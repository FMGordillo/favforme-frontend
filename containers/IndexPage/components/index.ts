export * from "./Actions";
export * from "./Alliances";
export * from "./CallToActions";
export * from "./Contact";
export * from "./IntroOne";
export * from "./IntroTwo";
export * from "./ODS";
export * from "./Partners";
export * from "./RSE";
