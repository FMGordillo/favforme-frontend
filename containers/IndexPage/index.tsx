import {
  ActionsComponent as Actions,
  AlliancesSection,
  CallToActionSection,
  ContactSection,
  IntroOne,
  RSE,
} from "./components";
import { Divider, Layout } from "@/components";
import { ActionIndex } from "@/pages";
import { NextPage } from "next";

interface IndexPageProps {
  actions: ActionIndex[];
}

export const IndexPage: NextPage<IndexPageProps> = ({ actions }) => {
  return (
    <Layout
      headProps={{
        title: "Inicio",
      }}
      header
      headerProps={{
        isIndex: true,
      }}
    >
      <Divider />
      <IntroOne />
      <Divider />
      <Actions actions={actions} loading={false} />
      <Divider />
      <CallToActionSection />
      <Divider />
      <RSE />
      <Divider />
      <AlliancesSection />
      <Divider />
      <ContactSection />
      <Divider />
    </Layout>
  );
};
