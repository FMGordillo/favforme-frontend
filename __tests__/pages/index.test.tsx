import { shallow } from "enzyme";
import IndexPage from "../../pages/index";

describe("Page /", () => {
  it("should render", () => {
    const tree = shallow(<IndexPage />);
    expect(tree).toMatchSnapshot();
  });
});
