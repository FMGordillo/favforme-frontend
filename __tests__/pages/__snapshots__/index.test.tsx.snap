// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page / should render 1`] = `ShallowWrapper {}`;
